packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -221,15 +222,17 @@ function createSoapBody({
   const receiverDeviceId = gateway.oid;
   const toUrl = gateway.url;
   const providerId = bodyData.principalCareProviderIds[0];
-  const homeCommunityId = bodyData.samlAttributes.homeCommunityId;
+  const homeCommunityId = requiresMetriportOidInsteadOfInitiatorOid(gateway)
```

Comment:
nit: can put in a shared func that gets the right OID, so that ternary isn't written twice